tests/lax_numpy_test.py

```diff
@@ -1306,6 +1306,10 @@ def testOperatorRound(self):
           ('reflect', nonempty_shapes),
           ('wrap', nonempty_shapes),
           ('edge', nonempty_shapes),
+          ('maximum', nonempty_shapes),
+          ('minimum', nonempty_shapes),
+          ('mean', nonempty_shapes),
+          ('median', nonempty_shapes),
```

Comment:
I'd lean toward splitting these into separate tests, because `constant_values` has no effect on these modes, and so this parametrization is generating ~10 identical tests for each. This would also allow separate parametrization over the `stat_length` argument, if/when it is implemented.